### PR TITLE
[BIP-525] Business Development Funding Proposal Q1-2024 and Retroactive Q4-2023 Funding

### PR DESCRIPTION
**Pull request by gosuto-inzasheru**
forum: https://forum.balancer.fi/t/bip-xxx-business-development-funding-proposal-q1-2024-and-retroactive-q4-2023-funding/5481/2
sim: https://dashboard.tenderly.co/public/safe/safe-apps/simulator/6e085de7-5cdb-480b-9750-024b4de871b0/

### CODE REVIEW

**Comment by gosuto-inzasheru** [comment]
`0xC7E84373FC63A17B5B22EBaF86219141B630cD7a` is not caught by the report atm, needs https://github.com/BalancerMaxis/bal_addresses/pull/170/files merged

File: action-scripts/brownie/scripts/report_gauges.py [hunk 1]
```diff
@@ -506,7 +506,11 @@ def parse_no_reports_report(
             contractMethod = transaction.get("contractMethod", {})
             no_reports.append(
                 {
-                    "fx_name": contractMethod.get("name", "!!N/A!!"),
+                    "fx_name": (
```
Comment (Tritium-VLK) [inline action-scripts/brownie/scripts/report_gauges.py:509]: What is this change about @gosuto-inzasheru.  I think that if/else was there for a reason and it's fragile.  Can we abstract this to another PR?

**Review by Tritium-VLK — CHANGES_REQUESTED** [review]

File: action-scripts/brownie/scripts/report_gauges.py (hunk 1 above)
Comment (gosuto-inzasheru) [reply action-scripts/brownie/scripts/report_gauges.py:509]: i still think it is an issue that these ci reports crash silently, it should be obvious that they didnt complete their run.

**Review by Tritium-VLK — APPROVED** [review]

**Review by Tritium-VLK — APPROVED** [review]

File: BIPs/2024-W2/BIP-525.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        "name": "transfer",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "to": "0xC7E84373FC63A17B5B22EBaF86219141B630cD7a",
+        "value": "20000000000"
+      }
+    },
+    {
+      "to": "0xC7E84373FC63A17B5B22EBaF86219141B630cD7a",
```
Comment (Tritium-VLK) [inline BIPs/2024-W2/BIP-525.json:49]: This does not seem like a complete transaction. Is it trying to be a transfer?

File: action-scripts/brownie/scripts/report_gauges.py (hunk 1 above)
Comment (Tritium-VLK) [reply action-scripts/brownie/scripts/report_gauges.py:509]: @gosuto-inzasheru I think the issue is that the payload is wrong and the second tx is a malformed transfer.   I guess it makes sense to have it finish and post an error or something, but maybe we should be louder about the fact that something is wrong with one of the transactions.  Can we abstract this into another PR and think about how to handle it.  There are some other calldata only use_cases we should think about/see how they interact.

**Review by Tritium-VLK — CHANGES_REQUESTED** [review]

**Review by Tritium-VLK — APPROVED** [review]

File: BIPs/2024-W2/BIP-525.json (hunk 2 above)
Comment (Tritium-VLK) [reply BIPs/2024-W2/BIP-525.json:49]: Spoke about this on a call.  THis is an ETH transfer. Created ticket to further improve reporting.

**Pull request merged**